api/src/pcapi/core/educational/api/dms.py

```diff
@@ -0,0 +1,165 @@
+from datetime import datetime
+from datetime import timedelta
+import logging
+import typing
+
+from pcapi import settings
+from pcapi.connectors.dms import api as dms_api
+from pcapi.connectors.dms import models as dms_models
+from pcapi.core.educational import models as educational_models
+from pcapi.core.offerers import models as offerers_models
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+
+
+def update_dms_status(procedure_id: int = 0) -> None:
```

Comment:
(question / détail) Est-ce qu'il y a un réel intérêt à fournir une valeur par défaut, en sachant qu'elle n'a aucune chance de fonctionner (puisqu'il n'y a pas de procédure DMS avec cet identifiant) ?

```suggestion
def update_dms_status(procedure_id: int) -> None:
```

Reply:
La valeur par defaut doit être fausse (False) et dans ce cas on fait tourner les 3 procedures. 
La logique est que par defaut (dans 99% des cas) on fera tourner les 3 procedures avec la cron et dans le cas ou procedure_number est setté (donc True) on fera passer que cette procedure la.

Reply:
corrigé avec corentin